cmake/SketchUpAPICpp.cmake

```diff
@@ -6,6 +6,7 @@ file(GLOB_RECURSE CPP_API_HEADERS ${CPP_API_INCLUDE_PATH}/*.hpp)
 file(GLOB_RECURSE CPP_API_SOURCES ${CPP_API_SOURCE_PATH}/*.cpp)
 
 add_library(SketchUpAPICpp STATIC ${CPP_API_HEADERS} ${CPP_API_SOURCES})
+target_link_libraries(SketchUpAPICpp LINK_PUBLIC SLAPI_LIB)
```

Comment:
I had added this to the test project, because the static lib under windows doesn't need to link SLAPI to compile.
Is this not the case under Xcode?

Reply:
Apologies, you are quire right, this is not needed even for Xcode